index.c

```diff
@@ -689,23 +690,25 @@ static void change_folder_mailbox(struct Menu *menu, struct Mailbox *m, int *old
   if (!m)
     return;
 
+  struct Mailbox *cm = ctx_mailbox(Context);
```

Comment:
Please can you rename this to `m_ctx`.
I've given all the "main" structs standard names:
- `struct Mailbox *m`
- `struct Email *e`
etc

Where there are multiple instances, I've added suffixes, e.g. `m_sel`, `m_cur`.